includes/definitions/discovery/apc.yaml

```diff
@@ -218,6 +250,13 @@ modules:
                     index: 'airIRRP100UnitStatusCompressorDriveDCLinkVoltage.{{ $index }}'
                     descr: Compressor Drive DC Link
                     divisor: 10
+                -
+                    oid: rPDU2PhaseStatusTable
+                    value: rPDU2PhaseStatusVoltage
```

Comment:
This might be redundant as we already fetch it in includes/discovery/sensors/voltage/apc.inc.php?